docs/en/1-Experiments/Remote-Desktop.md

```diff
@@ -1,5 +1,15 @@
 # Overview
 
+### Ubuntu Virtual Desktop
+
+_You can run a full [**Ubuntu Desktop**](https://ubuntu.com/desktop), with typical applications, right inside your browser, using [**Kubeflow!**](https://kubeflow.aaw.cloud.statcan.ca/)_
```

Comment:
can probably provide the base (rocker image) github link instead of ubuntu.com since that will actually show them what is installed on the image we offer, on top of the vanilla image they use there.